mods/persistence/controllers/subsystems/initialization/chargen.dm

```diff
@@ -1,13 +1,12 @@
-var/global/list/chargen_spawns = list()
+var/global/list/chargen_areas = list() //List of pod areas, and a number of times assigned was called on a given area for debugging purpose
+var/global/list/chargen_landmarks //List of all the chargen landmarks available for spawn.
```

Comment:
Having this as a LAZYLIST is unnecessary.

Reply:
Eh, its not like it makes much of a difference.

Reply:
For code cleanliness reasons I'd prefer it be a normal list, especially since this is a global list.